instrumentation/aws-lambda-1.0/library/src/main/java/io/opentelemetry/instrumentation/awslambda/v1_0/TracingRequestStreamHandler.java

```diff
@@ -86,11 +87,15 @@ protected abstract void doHandleRequest(InputStream input, OutputStream output,
 
     private final OutputStream delegate;
     private final io.opentelemetry.context.Context otelContext;
+    private final OpenTelemetrySdk openTelemetrySdk;
 
     private OutputStreamWrapper(
-        OutputStream delegate, io.opentelemetry.context.Context otelContext) {
+        OutputStream delegate,
+        io.opentelemetry.context.Context otelContext,
+        OpenTelemetrySdk openTelemetrySdk) {
```

Comment:
It's definitely *NOT* a good thing, but if the lambda instrumentation needs to call `forceFlush` it's the only option we have right now...unless we introduce a shim interface that exposes that function, and make people construct an implementation that hides the SDK.

Reply:
Yeah - it's not a good thing, but when discussing in spec repo and sig we decided that for now, lambda requiring the dependency is fine since it's the only way.

https://github.com/open-telemetry/opentelemetry-specification/issues/1287#issuecomment-745697250